charts/zeebe-benchmark/values.yaml

```diff
@@ -155,14 +121,16 @@ camunda-platform:
     # IoThreadCount defines how many threads can be used for the exporting on each broker pod
     ioThreadCount: 3  
 
-    # PodSecurityContext defines the security options the Zeebe broker pod should be run with
-    podSecurityContext:
-      # needed to make sure the zeebe user has file permissions on mounted PVCs
-      # TODO remove once it's configured in the upstream helmchart
-      fsGroup: 1000
-
     # ContainerSecurityContext defines the security options the Zeebe broker container should be run with
     containerSecurityContext:
+      ## @param securityContext.allowPrivilegeEscalation
+      allowPrivilegeEscalation: true
+      ## @param securityContext.privileged
+      privileged: true
+      ## @param securityContext.readOnlyRootFilesystem
+      readOnlyRootFilesystem: true
+      ## @param securityContext.runAsUser
```

Comment:
:upside_down_face: The comments are more confusing than anything imo, e.g. what is `@param securityContext.privileged`

Reply:
Yeah sorry I copied this from the camunda platform helm charts https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml#L414-L430

I think this was added to better generate the readme.